public/app/features/plugins/admin/state/hooks.ts

```diff
@@ -36,7 +36,8 @@ export const useGetAll = (filters: PluginFilters, sortBy: Sorters = Sorters.name
 
 export const useGetUpdatable = () => {
   const { isLoading } = useFetchStatus();
-  const updatablePlugins = useSelector(selectPlugins({ isInstalled: true, hasUpdate: true }));
+  const { plugins: installed } = useGetAll({ isInstalled: true });
+  const updatablePlugins = installed.filter((p) => isPluginUpdateable(p));
```

Comment:
Nit: you could just pass the function I think:
```suggestion
  const updatablePlugins = installed.filter(isPluginUpdateable);
```